gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -289,6 +272,84 @@ def run(self) -> GtsfmData:
             process_graph_generator.is_image_correspondence = True
         process_graph_generator.save_graph()
 
+        if self.parsed_args.run_low_quality:
+            # TODO(johnwlambert): Check if these are None, otherwise don't override.
+            COMBINATIONS = [(self.parsed_args.num_matched, self.parsed_args.max_frame_lookahead)]
+        else:
+            # Run multiple trials at different (num_matched, max_frame_lookahead) settings.
+            COMBINATIONS = [
+                (0, 5),
+                (0, 10),
+                (5, 0),
+                (5, 5),
+                (5, 10),
+                (10, 0),
+                (10, 5),
+                (10, 10),
+            ]
```

Comment:
I think allowing the user to put in a list of `num_matched` or `max_frame_lookahead` values would be cleaner (similar to what we do for multi-step BA), as opposed to hardcoding the combinations.

Reply:
I like this idea, but inputting a large list will be troublesome I think.